src/components/Contact/ContactCard2.jsx

```diff
@@ -1,59 +1,99 @@
 import * as React from "react";
-import CardContent from "@mui/material/CardContent";
-import Typography from "@mui/material/Typography";
-import {CardHeading, CardImage, CardSubheading, CustomCard} from "../mui/CardPackages";
-import {LockClock} from "@mui/icons-material";
+import { LockClock } from "@mui/icons-material";
+import {
+    ServiceItemContainer,
+    ServiceItemBox,
+    ServiceItemIconContainer,
+    ServiceItemIcon,
+    ServiceItemHeading,
+    ServiceItemDescription,
+} from "../../components/Home/ServicesOverview/ServiceOverviewPckgs";
+import { Box } from "@mui/material";
+import { useTheme } from "../../contexts/themeContext";
 
-export default function BasicCard() {
+export default function ContactCard2() {
+    const { theme } = useTheme();
     return (
-        <CustomCard>
-            <CardContent
+        <Box
+            sx={{
+                textAlign: "left !important",
+                padding: "1.5rem !important",
+                marginTop: "0 !important",
```

Comment:
Make the styles for both of these cards, ContactCard and ContactCard2 reusable and reduce wordiness, move them into the appropriate files since having them separate is useless

Reply:
Not needed, too much wordiness convolutes logic